cmd-replay.c

```diff
@@ -239,6 +239,13 @@ static void setup_field(struct opts *opts)
 		s++;
 	}
 
+	if (opts->range.start > 0) {
+		if (opts->range.start_elapsed)
+			add_field(field_table[5]);
+		else
+			add_field(field_table[3]);
+	}
```

Comment:
@namhyung Ah.. I missed the above comment you wrote 20 days ago...
I understood what you said. I missed the case the user want to see output on particular fields they assign.
Will fix it as v3 !